src/appShell/App/usageAgreements/GenieAgreement.tsx

```diff
@@ -8,12 +8,8 @@ export const GENIE_PERSISTENCE_KEY = 'genie-private-usage-agreement';
 export function shouldShowGenieWarning() {
     // we want to show a warning message on private cbioportal instances
     // to prevent users from adding links in manuscripts
-    const showGenieWarning = ['cbioportal-genie-private'].includes(
-        AppConfig.serverConfig.app_name!
-    );
-
     return (
-        showGenieWarning &&
+        AppConfig.serverConfig.app_is_private_portal &&
```

Comment:
Let's not change this one. The `cbioportal-genie-private` name is unique, and this code should always be specific to that server.